GraphWebApi/wwwroot/OpenApi.yaml

```
@@ -1,43 +1,48 @@
-openapi: 3.0.0
-info: 
-  title: "Graph Explorer API" 
-  version: "1.0.0" 
+openapi: 3.0.3
+info:
+  title: "DevX API"
+  version: "20210407.1"
```

Update version to the latest one 
```suggestion
  version: "20210415.1"
```